static/modules/gamemanager.js

```diff
@@ -10,20 +10,27 @@ import {
 import { inPlaceFilter } from "./helpers.js";
 import { isColliding } from "./collision.js";
 import { Vector } from "./vector.js";
+import { GuiElement } from "../modules/guielement.js";
 
 const BLUR_SCALAR = 2;
 
 class GameManager {
   updateTime = 10;
   overTime = 0;
 
+  /** @type {number} */
   totalTime = 0;
-
+  /** @type {number} */
+  gameTime = 0;
+  /** @type {number} */
   previousTime = 0;
 
   /** @type {Entity[]} */
   entities = [];
 
+  /** @type {GuiElement} */
```

Comment:
This is not type GuiElement. This is `Map<GuiElement>`
VSCode would give you a warning. There must be a way to get coc to do it but I haven't figured out how.